scripts/airdropBsxToHdx/getVestingSchedules.ts

```diff
@@ -0,0 +1,85 @@
+// ✅  -> calculate BSX reward for accounts from .json of getData
+
+// ✅ -> calulate 5 000 000 000 proportional to holdings at the time for each holder
+//  ->  output a .json schedule
+
+// ✅  -> use same schedule as we used for 70% of each contributor's BSX crowdloan rewards
+
+const BN = require("BN.js")
+const airdropData = require("./data/airdropData.json")
+
+type DynamicVestingInfo = {
+    destination: string,
+    schedule: {
+        amountToBeVested: string,
+        start: string,
+        period: string,
+        per_period: string,
+        period_count: string 
+    }
+}
+
+type VestingBatch = DynamicVestingInfo[]
+
+const airdropBsxAllocation = new BN('5000000000000000000000')
+// https://basiliskfi.substack.com/p/introducing-basilisk 
+// ^^^ 25% of HDX holders ( HDX LBP participants, team and investors ) allocation
+const sumHdxBalances = new BN(airdropData.totalAllOgHdxBalances) 
+const bsxAirdroppedPerOneHdx = 
+    airdropBsxAllocation
+    .div(sumHdxBalances)
+    .toString()
+
+console.log('\n amount of BSX tokens you get per one HDX token: ', bsxAirdroppedPerOneHdx + '\n') // --> 10
+
+const leaseStartBlock = new BN('9334719').toString()
+    // our auction ended on this block 
+    // ^^^ _9 334 719_ https://kusama.subscan.io/auction/8
+
+const vestScheduleEndBlock = new BN('13834719')
```

Comment:
final block has to be ~22 months after start

Reply:
please review latest commit for implementation of this @lumir-mrkva thanks!